readthedocs/index.rst

```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
 
 .. code-block:: python
 
+   import sys
+   import os
+   os.chdir(sys.path[0])
```

Comment:
Why is this needed, and what path is exactly that?

Reply:
It is to set the working directory to the script's path and later delete the `.session` file